### PR TITLE
Docs: Small fixes for Template Variables Doc

### PR DESCRIPTION
**Pull request by sarah-spang**
**Which issue(s) does this PR fix?**:
- Update list of datasources that support ad hoc variables so it's consistent throughout doc
- Fixes some phrasing and formatting errors
- Add deprecation note about SingleStat panel

### CODE REVIEW

File: docs/sources/dashboards/variables/add-template-variables/index.md [hunk 1]
```diff
@@ -281,6 +281,8 @@ This variable is the `$__interval` variable in milliseconds, not a time interval
 
 This variable is only available in the Singlestat panel and can be used in the prefix or suffix fields on the Options tab. The variable will be replaced with the series name or alias.
 
+> **Note:** Singestat panel is no longer available from Grafana 8.0
```
Comment (imatwawana) [inline docs/sources/dashboards/variables/add-template-variables/index.md:284]:
```suggestion
> **Note:** The Singlestat panel is no longer available from Grafana 8.0. 
```

**Review by imatwawana — APPROVED** [review]
Thanks for the contribution! This looks good! If you don't have merge powers, just ping me so I can have someone do that for you.

**Comment by sarah-spang** [comment]
@imatwawana Thanks for reviewing! Committed your suggested change, but I don't have the power to merge.

**Comment by imatwawana** [comment]
> @imatwawana Thanks for reviewing! Committed your suggested change, but I don't have the power to merge.

@chri2547 - can you merge when you get a minute?

**Comment by chri2547** [comment]
@sarah-spang , @imatwawana  - it looks like we need Sarah to sign the CLA? That particular check is still in limbo.

**Comment by sarah-spang** [comment]
@chri2547 I've signed the CLA in the past and have submitted a few other PRs in this repo recently that were merged. I don't see any prompt for me to sign it again, so not sure why it's hanging on that check. If there's something else I need to do, please let me know.

**Comment by chri2547** [comment]
> @chri2547 I've signed the CLA in the past and have submitted a few other PRs in this repo recently that were merged. I don't see any prompt for me to sign it again, so not sure why it's hanging on that check. If there's something else I need to do, please let me know.

@sarah-spang , I also thought you'd signed the CLA. Ok, I'll investigate on my end. Thanks!

**Pull request merged**